Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -76,6 +77,8 @@ public string ElementTitle
             set { mElementTitle = value; }
         }
 
+        public ObservableList<ActInputValue> actInputValue = new ObservableList<ActInputValue>();
```

Comment:
I have got better idea. Instead of list list if input values let's create object Act and then we don't need to write the functions for add and get param again.